### PR TITLE
master-tv migration to v6

### PR DESCRIPTION
**Pull request by serge-sky**
Thanks for opening a PR!
Since this is a volunteer project and is very active, anything you can do to reduce the amount of time needed to review and merge your PR is appreciated.
The following steps will help get your PR merged quickly:

#### Update the documentation
If you've added new functionality, update the README.md with an entry for your prop or event.
The entry should be inserted in alphabetical order.

#### Update the changelog
After you open the PR, update the CHANGELOG.md file with an entry pointing to your PR.

#### Provide an example of how to test the change
If the PR requires special testing setup provide all the relevant instructions and files. This may include a sample video file or URL, configuration, or setup steps.

#### Focus the PR on only one area
Testing multiple features takes longer than isolated changes and if there is a bug in one feature, prevents the other parts of your PR from getting merged until it gets fixed.
If you're touching multiple different areas that aren't related, break the changes up into multiple PRs.

#### Describe the changes
Add a note describing what your PR does. If there is a change to the behavior of the code, explain why it needs to be updated.

### CODE REVIEW

File: android/src/main/java/com/brentvatne/exoplayer/VideoEventEmitter.java [hunk 1]
```diff
@@ -174,11 +267,24 @@ void load(double duration, double currentPosition, int videoWidth, int videoHeig
...
+        event.putArray(field, array);
+        return event;
+    }
+
+    public void audioTracks(ArrayList<Track> audioTracks){
+        receiveEvent(EVENT_AUDIO_TRACKS, arrayToObject(EVENT_PROP_AUDIO_TRACKS, audioTracksToArray(audioTracks)));
+    }
+
+    public void textTracks(ArrayList<Track> textTracks){
+        receiveEvent(EVENT_TEXT_TRACKS, arrayToObject(EVENT_PROP_TEXT_TRACKS, textTracksToArray(textTracks)));
```
Comment (sharan-SKY) [inline android/src/main/java/com/brentvatne/exoplayer/VideoEventEmitter.java:283]: seems like its simplified, good one

**Comment by sharan-SKY** [comment]
@serge-sky looks like none of the youbora changes were ported to the new code?

**Comment by sharan-SKY** [comment]
Also your changes around `if (isBehindLiveWindow(e) || e.type == ExoPlaybackException.TYPE_RENDERER) {` to fix black screen issue did not make it through too

**Comment by serge-sky** [comment]
> @serge-sky looks like none of the youbora changes were ported to the new code?

it was, please check ReactExoplayerView.java

**Comment by serge-sky** [comment]
> Also your changes around `if (isBehindLiveWindow(e) || e.type == ExoPlaybackException.TYPE_RENDERER) {` to fix black screen issue did not make it through too

MediaCodecVideoRenderer error is supposed to be fixed in the latest versions of Exoplayer, but it's still for us to verify

**Comment by sharan-SKY** [comment]
all good, just checked locally, github PR UI is broken for big as PRs

**Review by sharan-SKY — APPROVED** [review]

**Comment by sharan-SKY** [comment]
great job 👏

**Pull request merged**